beacon-chain/rpc/service.go

```diff
@@ -310,7 +309,7 @@ func NewService(ctx context.Context, cfg *Config) *Service {
 		s.cfg.Router.HandleFunc(
 			e.template,
 			e.handlerWithMiddleware(),
-		).Methods(e.methods...)
```

Comment:
You removed `Methods` but there is no replacement

Reply:
Actually now it is done like router.handleFunc("GET <endpoint-url>",handler);

Reply:
this is still an issue, you can't use the e.template directly, it doesn't have the methods in the path

Reply:
What's the alternative then ?